pages/index.js

```diff
@@ -97,6 +97,77 @@ const Home = () => (
           link="https://blog.commit.dev/articles/streaming-out-the-monolith"
         />
       </div>
+      <hr className={styles.sectionSeparator} />
+      <ol className={styles.timeline}>
+        <li className={styles.timelineItem}>
+          <div>
+            <Text className={styles.date} sectionText>
+              2019
+            </Text>
+            <Text className={styles.event}>Founded by Software Engineers</Text>
+          </div>
+        </li>
+        <li className={styles.timelineItem}>
+          <div>
+            <Text className={styles.date} sectionText>
+              2020
+            </Text>
+            <Text className={styles.event}>
+              We grew <span className={styles.boldPink}>370%</span>
+            </Text>
+          </div>
+        </li>
+        <li className={styles.timelineItem}>
+          <div>
+            <Text className={styles.date} sectionText>
+              2021
+            </Text>
+            <Text className={styles.event}>
+              We aim to hire{' '}
+              <span className={styles.boldPink}>500 Engineers</span> with an
+              emphasis on <span className={styles.boldPink}>diversity</span>
+            </Text>
+          </div>
+        </li>
+      </ol>
+      <div className={styles.whyCommit}>
```

Comment:
similar to the timeline, could have own file in the `/sections/` folder.

Reply:
Addressed